src/runtime_src/core/tools/common/XBUtilities.cpp

```diff
@@ -357,6 +358,19 @@ XBUtilities::get_available_devices(bool inUserDomain)
       } catch(...) {
         // The id wasn't added
       }
+
+     try {
+       std::string stream;
+       auto  instance = xrt_core::device_query<xrt_core::query::instance>(device);
+       if(device->is_userpf())
+         stream = "user";
+       else
+         stream = "mgmt";
```

Comment:
This is still not fixed as requested:
`std::string pf = device->is_userpf() ? "user" : "mgmt";`